### ISSUE TITLE
Websocket doesn't connect

### ISSUE THREAD

**Issue by m00nm00nm00n** [discord-php/DiscordPHP#638]
This is all I get when I run my bot:

[2021-12-12T02:26:54.443059+00:00] DiscordPHP.INFO: gateway retrieved and set {"gateway":"wss://gateway.discord.gg/?v=6&encoding=json","session":{"total":1000,"remaining":1000,"reset_after":0,"max_concurrency":1}} []
[2021-12-12T02:26:54.443944+00:00] DiscordPHP.INFO: starting connection to websocket {"gateway":"wss://gateway.discord.gg/?v=6&encoding=json"} []

after this it just returns to the command prompt...

what am I doing wrong? seems like the connection doesnt complete

ive added the bot to my server and sent it DMs in Discord.. nothing shows up in the CLI

**Comment by SQKo** [comment]
Recent composer dependencies update caused this

It has been fixed in 4e35fa2 so please update to `dev-master` branch

Looking at the gateway URL which is still v6 It seems that you are still using older DiscordPHP version `5.1.x` or lower, if you are not able to upgrade now, please bump the `ratchet/pawl` to at least version `^0.4.1`

**Comment by davidcole1340** [comment]
As said above previous versions of Pawl aren't compatible with the newer versions of ReactPHP.

I have updated the Pawl version in Version 6.x and the latest `master` branch of DiscordPHP but won't be updating for any earlier versions, so you'll need to override it in your `composer.json`:

```json
{
  "require": {
    "team-reflex/discord-php": "^5.1",
    "ratchet/pawl": "0.4.1 as 0.3.0"
  }
}
```

However I strongly recommend you upgrade to the latest version of DiscordPHP (ideally `master` branch) as Discord gateway v6 is deprecated and will be discontinued soon (so you'll be unable to run your bot).
